file_paths.md

```diff
@@ -163,39 +163,107 @@ and `pxp-agent`, all running as `LocalSystem` by default. It will not
-        modules *                             # user modulepath
+        hiera.yaml                            # :hiera_config
+        modules                               # user modulepath
+
+    C:\ProgramData\PuppetLabs\mcollective *
+        client.cfg *
+        facts.yaml *
+        server.cfg *
 
-    C:\ProgramData\PuppetLabs\facter *                                       same
+    C:\ProgramData\PuppetLabs\puppet          # :confdir
+        auth.conf *                           # :rest_authconfig
+        autosign.conf                         # :autosign
+        binder_config.yaml                    # :binder_config
+        csr_attributes.yaml                   # :csr_attributes
+        custom_trusted_oid_mapping.yaml       # :trusted_oid_mapping_file
+        device.conf                           # :deviceconfig
+        fileserver.conf                       # :fileserverconfig
+        puppet.conf *                         # :config
+        routes.yaml                           # :route_file
+        ssl                                   # :ssldir
+
+    C:\ProgramData\PuppetLabs\pxp-agent
+        modules *                             # stores configuration files for pxp-agent modules
+            pxp-module-puppet.conf *          # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
+        pxp-agent.conf                        # pxp-agent configuration file
+
+    C:\Program Files\Puppet Labs\bin *
+        environment.bat *                     # setup LOAD_PATH
+        puppet.bat *                          # bat file wrappers
+        hiera.bat *
+        mco.bat *
+        facter.bat *
+        puppet_shell.bat *                    # targets for shortcuts
+        run_facter_interactive.bat *
+        run_puppet_interactive.bat *
+
+    C:\Program Files\Puppet Labs\facter *
         facts.d *                             # external facts directory (not pluginsync'ed)
 
-    C:\ProgramData\PuppetLabs\mcollective
+    C:\Program Files\Puppet Labs\mcollective *
         plugins *                             # user installed plugins
-        etc *                                                                same
-            client.cfg
-            facts.yaml
-            server.cfg
-        var *                                                                same
-            log *
-                mcollective.log
 
-    C:\ProgramData\PuppetLabs\puppet
-    -------------------------------------------------------------------------------------------------------
-        cache                                 # :vardir                      C:\ProgramData\PuppetLabs\puppet\var
+    C:\Program Files\Puppet Labs\puppet *     # ruby-puppet root
+        bin *                                 # executables and dlls
+            c_rehash *
+            catstomp *
+            catstomp.bat *
+            curl.exe *
+            curl-config *
+            daemon.bat ?
```

Comment:
So, in the list of files from https://github.com/puppetlabs/marionette-collective/tree/master/ext/windows, we only need mco.bat? Everything else shouldn't be necessary except for environment.bat, but we'll have a project-level version of that file that will work with everything (a la puppet_for_the_win). Does that sounds right?